regtests/bin/matrix.base

```diff
@@ -1903,6 +1904,12 @@
     echo "$rtst -s MPI_OMPH -w work_MPI_OMPH -f -p $mpi -n $nr -t $nth $ww3 -o both ww3_tp2.16" >> matrix.body
   fi
 
+  if [ "$multi09" = 'y' ]
+  then
+    echo ' ' >> matrix.body
+    echo "$rtst -s MPI -w work_MPI -m grdset_a -f -p $mpi -n $np $ww3 mww3_test_09" >> matrix.body
```

Comment:
"-N" is missing for namelist
echo "$rtst -s MPI -w work_MPI -m grdset_a -f -p $mpi -n $np -N $ww3 mww3_test_09" >> matrix.body

Reply:
@aliabdolali I thought it was normal to not specify the -N switch so that the regression test user can set it via the $rtst variable. I.e. rtst="run_test -N".

I am happy to specify it explicitly in the matrix.base if you prefer though (I guess we are deprecating the .inp files...)

Reply:
You are right, however, I was not able to run it with ww3_grid.<>.nml files. The run_test did not recognize them and it failed. Have you tested them at your end without -N?

Reply:
So - the NML option (-N) works for me and the test runs ok (once I had installed the data using ww3_from_ftp).
I thought I had set up .inp files for the ww3_grid program, but perhaps not. Would you like me to create these, considering that we are deprecating .inp files?

Reply:
> So - the NML option (-N) works for me and the test runs ok (once I had installed the data using ww3_from_ftp).
> I thought I had set up .inp files for the ww3_grid program, but perhaps not. Would you like me to create these, considering that we are deprecating .inp files?

So we either need to specifiy -N in matrix or add grid<>.inp files there. Since we do not have -N in the matrix for none of other tests, I would prefer to have *inp files for this one as well.

Reply:
OK - I will prepare some .inp files.

Reply:
@aliabdolali ww3_grid<>.inp files have been added for mww3_test_09.
I've tested running with both the .nml and .inp files and both are running ok for me.